src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in

```diff
@@ -206,10 +206,7 @@
     },
     "Microsoft.NET.Runtime.WebAssembly.Templates.${NetVersion}": {
       "kind": "template",
-      "version": "${PackageVersion}",
-      "alias-to": {
-          "any": "Microsoft.NET.Runtime.WebAssembly.Templates"
-      }
+      "version": "${PackageVersion}"
```

Comment:
@lewing Is this correct? We need the package id to be `Microsoft.NET.Runtime.WebAssembly.Templates.net9` (10, etc)

Reply:
Maui seems to have it the same https://github.com/dotnet/maui/blob/e4ac65895978e40527bac8e09bdc688e5046aa55/src/Workload/Microsoft.NET.Sdk.Maui.Manifest/WorkloadManifest.in.json